x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/inference_pipeline_card.tsx

```diff
@@ -217,6 +182,49 @@ export const InferencePipelineCard: React.FC<InferencePipeline> = (pipeline) =>
           </EuiText>
         </EuiConfirmModal>
       )}
+    </>
+  );
+};
+
+export const InferencePipelineCard: React.FC<InferencePipeline> = (pipeline) => {
+  const { modelId, pipelineName, types: modelTypes } = pipeline;
+  const modelType = getMLType(modelTypes);
+  const modelTitle = getModelDisplayTitle(modelType);
+  const isSmallScreen = useIsWithinMaxBreakpoint('s');
+
+  // TODO: Handle redacted model ID
```

Comment:
Added this TODO as reminder to handle this in a follow-up PR

Reply:
Is there a followup issue? Maybe we could replace the TODO with that before merge?

Reply:
Removed the TODO